### PR TITLE
fix(TFD-4512): Export minItems and maxItems as integer for PropertiesList.

### PR DESCRIPTION
**Pull request by pbailly**
**What is the problem this Pull Request is trying to solve?**
This inconsistency between the PropertiesList and other properties break the validation of the JSONSchema on the new version of the UI.
 
**What is the chosen solution to this problem?**
Export minItems and maxItems as Integer instead of string.
 
**Link to the JIRA issue**

 
**Please check if the Pull Request fulfills these requirements**
- [X] The PR commit message follows our [guidelines](../CONTRIBUTING.md)
- [X] Tests for the changes have been added (for bug fixes / features, coverage should be over 75% in the new code)
- [X] Docs have been added / updated (for bug fixes / features)
- [x] Related design / discussions / pages (not in JIRA), if any, are all linked or available in the Pull Request

 
**[ ] This Pull Request introduces a breaking change**

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/26/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/26/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/27/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/27/)

File: daikon/src/main/java/org/talend/daikon/serialize/jsonschema/JsonSchemaGenerator.java [hunk 1]
```diff
@@ -73,8 +73,8 @@ private void computeSchemaType(Properties cProperties, String formName, boolean
             // Override the title to always use the property display name.  This is necessary because PropertyList UI
             // representations use the title to describe the type of element it contains.
             schema.put(JsonSchemaConstants.TAG_TITLE, cProperties.getDisplayName());
-            schema.put(JsonSchemaConstants.TAG_MIN_ITEMS, ((PropertiesList<?>) cProperties).getMinItems());
-            schema.put(JsonSchemaConstants.TAG_MAX_ITEMS, ((PropertiesList<?>) cProperties).getMaxItems());
+            schema.put(JsonSchemaConstants.TAG_MIN_ITEMS, Integer.parseInt(((PropertiesList<?>) cProperties).getMinItems()));
```
Comment (fhuaulme) [inline daikon/src/main/java/org/talend/daikon/serialize/jsonschema/JsonSchemaGenerator.java]: Integer.parseInt(...) may throw a NumberFormatException if string is null. 
```
if (s == null) {
   throw new NumberFormatException("null");
}
```
Is it worth it to protect the call to parseInt()?

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/33/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/33/)

**Comment by pbailly** [comment]
This PR needed a validation from the studio team before merge.
But we did a UI fix instead, so this is not relevant anymore I close it.

**Pull request closed without merging**